app/controllers/deliveries/new/select-project.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+
+const PROJECT_ADMIN_AUTH_ROLE = 'project_admin';
+
+export default Ember.Controller.extend({
+  project: null,
+  application: Ember.inject.controller(),
+  currentDukeDsUser: Ember.computed.alias('application.currentDukeDsUser'),
+  currentUserProjectPermissions: Ember.computed('project', 'currentDukeDsUser.id', function () {
+    const projectId = this.get('project.id');
+    const currentUserId = this.get('currentDukeDsUser.id');
+    if (!projectId || !currentUserId) {
+      return [];
+    }
+    return this.get('store').query('duke-ds-project-permission', {
+      'project': projectId,
+      'user': currentUserId,
```

Comment:
The user parameter is added to the end of the url and filters the permissions to just hose for the current user. Otherwise the `firstObject` might be another user's permissions.

Reply:
I see, so my understanding of the urlForQuery method wasn’t complete. This query parameter still gets passed through

Reply:
Yep. That's why I had to `delete query.project;` when building the query or it ended up being used twice.